pkg/engine/resolve/resolve.go

```diff
@@ -1043,18 +1043,21 @@ func (r *Resolver) resolveObject(ctx *Context, object *Object, data []byte, obje
 
 	typeNameSkip := false
 	first := true
+	skipCount := 0
 	for i := range object.Fields {
 
 		if object.Fields[i].SkipDirectiveDefined {
 			skip, err := jsonparser.GetBoolean(ctx.Variables, object.Fields[i].SkipVariableName)
 			if err == nil && skip {
+				skipCount++
 				continue
 			}
 		}
 
 		if object.Fields[i].IncludeDirectiveDefined {
 			include, err := jsonparser.GetBoolean(ctx.Variables, object.Fields[i].IncludeVariableName)
 			if err != nil || !include {
+				skipCount++
```

Comment:
should we have here `err == nil` like with a skip?

Reply:
no, include is the other way around as skip
if include is true, err should be nil